index.js

```diff
@@ -4,7 +4,49 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+
+//const parameters = process.argv;
+let i;
+let result = 0;
+function calculator() {
+   
+if (process.argv.length == 2 ) {
```

Comment:
If i run `node run.js 1 2 3`, I get `undefined`. How can you update this conditional to make sure you get the correct error message?